src/components/checkbox/checkbox.scss

```diff
@@ -359,6 +357,13 @@ md-checkbox {
   }
 }
 
+
+.md-checkbox-unchecked {
+  .md-checkbox-background {
+    background-color: none; 
```

Comment:
I don't believe `none` is a valid value for `background-color`. Did you mean` transparent`?